server/graphql_schemas/attend/schema.py

```diff
@@ -30,14 +33,20 @@ def mutate_and_get_payload(cls, root, info, **input):
         status = input.get('status')
         db_event_id = from_global_id(event_id)[1]
         event = Event.objects.get(id=db_event_id)
+        event_date = datetime.strptime(event.start_date+'00', '%Y-%m-%d %H:%M:%S.%f%z')
+        today = timezone.now()
         user = info.context.user
         andela_user_profile = AndelaUserProfile.objects.get(
             user_id=user.id)
         try:
-            user_attendance = Attend.objects.get(
-                user=andela_user_profile, event=event)
-            user_attendance.status = status
-            user_attendance.save()
+            if today < event_date:
+                user_attendance = Attend.objects.get(
+                    user=andela_user_profile, event=event)
+                user_attendance.status = status
+                user_attendance.save()
+            else:
+                raise GraphQLError(
+                    "The event is nolonger available")
```

Comment:
The error message here appears to have a typo. I think we need a space between `no` and `longer`. So we have: `This event is no longer available`